docs/pages/docs/core-concepts/caching.mdx

```diff
@@ -201,22 +199,21 @@ As of 1.4, when you run `turbo run build`, Turborepo will not consider any build
 
 ### `eslint-config-turbo`
 
-To help detect additional places where unseen dependencies can be creeping into your builds, you can use `eslint-config-turbo` for further in-editor assistance to help ensure your Turborepo cache can be correctly shared across every environment. While automatic environment variable inclusion should cover most situations with most frameworks, this ESLint config will provide just-in-time feedback for teams using other build time inlined environment variables that are not framework-prefixed but impact build outputs. This will also help support teams using their own in-house frameworks that we cannot detect automatically.
+To further assist in detecting unseen dependencies creeping into your builds, and to help ensure that your Turborepo cache is correctly shared across every environment, use the [`eslint-config-turbo`](https://www.npmjs.com/package/eslint-config-turbo) package. While automatic environment variable inclusion should cover most situations with most frameworks, this ESLint config will provide just-in-time feedback for teams using other build time inlined environment variables. These variables are not framework-prefixed but impact build outputs. This will also help support teams using their in-house frameworks that we cannot detect automatically.
```

Comment:
```suggestion
To further assist in detecting unseen dependencies creeping into your builds, and to help ensure that your Turborepo cache is correctly shared across every environment, use the [`eslint-config-turbo`](https://www.npmjs.com/package/eslint-config-turbo) package. While automatic environment variable inclusion should cover most situations with most frameworks, this ESLint config will provide just-in-time feedback for teams using other build time inlined environment variables. This will also help support teams using in-house frameworks that we cannot detect automatically.
```